wing.py

```diff
@@ -593,7 +593,7 @@ def setup(self, wing, state, **kwargs):
 #                  'R_{req}': 500,
 #                  'CruiseAlt': ('sweep', np.linspace(20000,40000,8)),
 #                  'numeng': 1,
-#      ##            'W_{Load_max}': 6664,
+#      ##            'W_{Load_{max}}': 6664,
 #                  'W_{pass}': 91 * 9.81,
 #                  'n_{pass}': 150,
 #                  'pax_{area}': 1,
```

Comment:
@1ozturkbe are there any more changes?

Reply:
There are some bugs, trying to resolve. VSP coming out weird. Will let you know.

Reply:
Oh god, I think your last merge was problematic. Some constraints got moved/erased. Will resolve.

Reply:
The TE constraints got messed up....

Reply:
@1ozturkbe when where they messed up?

Reply:
When I automerged with your code just now.

Reply:
Also, I fixed it, but I'm still getting some really odd results because the engines are absurdly heavy. As in, the engines are pretty much next to the tail... :P

Reply:
@1ozturkbe post an example I don't understand

Reply:
@1ozturkbe whoops I misread that. I see what you're saying, still push some examples.

Reply:
![d8wonky](https://user-images.githubusercontent.com/15684191/29726444-c16f4254-899d-11e7-80aa-ce26ae345664.png)

Reply:
It makes kind of sense, but our engine weight is 83% higher than TASOPT, so it seems to contribute to a wing farther aft. Why is that?

Reply:
@1ozturkbe the 737 engine isn't that heavy...only 7,000lbf

Reply:
@1ozturkbe show a 737

Reply:
@1ozturkbe the engine isn't 84% heavy somehow there is a random 2x multiplier on engine weight in the percent diff file lol..

Reply:
@1ozturkbe the D8 engine is actually light. look at the latest push.

Reply:
But I think in TASOPT, Wengsys is for both engines? Also, will post 737 momentarily.

Reply:
I already removed the 2

Reply:
![737](https://user-images.githubusercontent.com/15684191/29726796-3074deb0-899f-11e7-8290-b417afde3101.png)

Reply:
And in TASOPT, Wengsys is for both engines, so the percent diff makes sense. Which means we are super heavy on the engines. Why?

Reply:
@1ozturkbe where do you see Wengsys is for both engines

Reply:
in TASOPT that is

Reply:
the 737 looks fine

Reply:
all i see in TASOPT is that Wengsys is the full weight of the engine, nacelle, and pylon. It is not physical for two engines, nacelles, and pylons to weal 11,000 pounds. Dry weight of a 737 engine is about 6000 pounds without nacelles or pylons.

Reply:
If you add up the weights, it is...

Reply:
Hmmm, isn't that odd?

Reply:
@1ozturkbe it's odd but I'm confident out engine is not that heavy

Reply:
@1ozturkbe for the D8 'x_w' is only 64% of the fuselage length. In this image the wing is at ~75%. are you sure this isn't a VSP error with the D8

Reply:
I really think this could be something to do with the D8 vsp...the 737 you just posted looks almost identical to the old vsp figs

Reply:
I checked the VSP. It is correct.

Reply:
wait wait...could have something to do with wing pitching moment

Reply:
noice, that's what I'm talking about! Lmk if it makes a difference.

Reply:
wait look at this http://www.aurora.aero/wp-content/uploads/2016/06/D8_Brochure.pdf

Reply:
`sol = run_optimal_D8('W_{f_{total}}', True, False)`